tools-and-examples/auto_update_exaop/auto_update_exaop.py

```diff
@@ -1,67 +1,87 @@
 # -*- coding: utf-8 -*-
-import xmlrpclib, ssl, time, os, sys
+import xmlrpc.client, ssl, time, os, sys
```

Comment:
Would be good to know the intention behind replacing `xmlrpclib` with `xmlrpc.client`...

Reply:
xmlrpc.client is the standard lib in Python3: [https://docs.python.org/3/library/xmlrpc.html.](https://docs.python.org/3/library/xmlrpc.html.)